src/plugins/condition/ConditionSetTelemetryProvider.js

```diff
@@ -21,9 +43,16 @@ export default class ConditionSetTelemetryProvider {
         let conditionManager = new ConditionManager(domainObject, this.openmct);
         conditionManager.on('conditionSetResultUpdated', callback);
 
+        let stopObservingForChanges = this.openmct.objects.observe(domainObject, 'configuration.conditionCollection', (newConditionCollection) => {
+            conditionManager.update(newConditionCollection);
+        });
+
         return function unsubscribe() {
             conditionManager.off('conditionSetResultUpdated');
             conditionManager.destroy();
+            if (stopObservingForChanges) {
+                stopObservingForChanges();
+            }
```

Comment:
This is the only way to use conditionSet telemetry to show the output from ConditionSet.vue since the condition set domainObject is actively changing.
Alternately we can change conditionSet.vue to *not* use telemetry and just handle the `conditionSetResultUpdated` event instead.

Reply:
Hmmm. I'd like to get more eyes on this one.

@deeptailer, @akhenry, any guidance?

Reply:
I've fixed in #2748. There is probably a better way to do this then to destroy listeners (and other stuff) and then recreate them with the mutated DO, but this works based on the existing code. We can refactor after merged in.